docs/components/input/index.html

```diff
@@ -0,0 +1,40 @@
+<div class="input-component">
+  <h1>Input</h1>
+  <p class="text-lead margin-vertical-2">
+    Get user input using these input fields.
+  </p>
+
+  <h2 class="margin-vertical-3">Normal Input Types</h2>
+  <p>
+    This is the Example of basic input types used in a form with validation and
+    error message.
+  </p>
+  <div class="input-container margin-vertical-4">
+    <form action="javascrpt:;" class="form">
+      <div class="margin-bottom-3">
+        <label for="TxtEmail" class="form-label margin-vertical-2"
```

Comment:
Minute suggestion :- HTML structure can be more good , you can use prettify or beautify(these both are VS code extension)

for example:-
Email address</label
>

this can be :-
Email Address</label>